src/components/Nav/Nav.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import Navcategory from './Navcategory';
+import './Nav.scss';
+
+const Nav = () => {
+  const [data, setData] = useState([]);
+  const [tab, setTab] = useState('nav-menu-listHover');
+  useEffect(() => {
+    fetch('/data/data.json')
+      .then(res => res.json())
+      .then(res => setData(res));
+  }, []);
+  const hover = e => {
+    setTab('nav-menu-list');
+  };
+  const leave = e => {
+    setTab('nav-menu-list-hover');
+  };
+  return (
+    <div className="nav">
+      <div className="nav-box">
+        <div className="nav-center">
+          <div className="nav-top">
+            <div className="logo">WESH</div>
+            <ul className="category">
+              <li className="li" onMouseEnter={hover}>
+                제품
+              </li>
+              <li className="li">위쉬소개</li>
+              <li className="li">매장안내</li>
+              <li className="li">스파</li>
+              <li className="li">이벤트</li>
+            </ul>
+            <div className="nav-right">
+              <img src="https://www.lush.co.kr/content/renewal/pc/images/ico/search.svg" />
+              <img src="https://www.lush.co.kr/content/renewal/pc/images/ico/bag.svg" />
+              <img src="https://www.lush.co.kr/content/renewal/pc/images/ico/account.svg" />
+            </div>
+          </div>
+          <div className={tab} onMouseEnter={hover} onMouseLeave={leave}>
+            <ul className="nav-menu">
+              <Link to="/">
+                <li className="li-list">전체상품</li>
+              </Link>
```

Comment:
```suggestion
<li>
  <Link to="/">전체상품</Link>
</li>
```
이 구조가 더 맞을 것 같은데 확인해주세요!